src/elasticsearch/index.ts

```diff
@@ -137,3 +137,17 @@ export const getIndexSettings = async (
   })) as SettingsResponse;
   return response;
 };
+
+export const setIndexWritable = async (
+  esClient: Client,
+  indexName: string,
+  enabled: boolean
+): Promise<SettingsResponse> =>
+  await esClient.indices.putSettings({
+    index: indexName.toLowerCase(),
+    body: {
+      settings: {
+        "index.blocks.write": enabled,
```

Comment:
The code I copied had `"index.blocks.write": "true",` but here `enabled` is a boolean not a string. I need to run this end-to-end before we merge this to ensure this functions the same (though I believe it will).

Reply:
I've run this end to end, working without errors.